MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -92,6 +96,13 @@ private static GamePadState PlatformGetState(int index, GamePadDeadZone deadZone
                 (state.Buttons.HasFlag(WGI.GamepadButtons.X) ? Buttons.X : 0) |
                 (state.Buttons.HasFlag(WGI.GamepadButtons.Y) ? Buttons.Y : 0) |
                 0;
+
+            // Check triggers
```

Comment:
Just take that as a note for your next PR @jackdolabany .  This is fine for now.